frontend/src/hooks/useWorkSheets.js

```diff
@@ -2,14 +2,19 @@ import { useState, useEffect, useCallback } from 'react'
 import { getWorkSheets } from '../api/WorkSheetAPI'
 
 export default function useWorkSheets() {
+  const [status, setStatus] = useState('CREATED')
+
   const [workSheets, setWorkSheets] = useState()
 
-  const updateWorkSheets = useCallback(async function () {
-    const { data } = await getWorkSheets()
-    if (data) {
-      setWorkSheets(data)
-    }
-  }, [])
+  const updateWorkSheets = useCallback(
+    async function () {
+      const { data } = await getWorkSheets(status)
+      if (data) {
+        setWorkSheets(data)
```

Comment:
Ez nem warningol? Elvileg a setWorkSheets is kellene a függőségbe.

Reply:
setWorkSheets hozzáadva a useEffect függőségeibe. Erre gondoltál ugye?

Reply:
igen, elvileg warningnak is kellene lennie a console-ban enélkül.

Reply:
Most látom csak, ennek itt kellene függőségnek lennie, nem a useEffect-nél.

Reply:
True, true, benéztem. Javítottam.